### ISSUE TITLE
Request alternative save file path Wii system 

### ISSUE THREAD

**Issue by Catmansys** [mgba-emu/mgba#3383]

**Comment by endrift** [comment]
It has this already but it requires manually editing the config file. It'd be way better if it could be edited from the config menu of course.

**Comment by Catmansys** [comment]
> It has this already but it requires manually editing the config file. It'd be way better if it could be edited from the config menu of course.

Oh really? Please teach me. I open the config int.txt but can’t fine the line which mentions save location.

**Comment by endrift** [comment]
Yeah, the config file isn't fully populated by default. You need to set the `savegamePath` value in the `[ports.wii]` section. Also available are `savestatePath`, `screenshotPath`, `patchPath`, and `cheatsPath`. I do need to add a folder selector though.

**Comment by askotx** [comment]
Would it be possible to add some defaults paths in code, like: (sd/usb):/mgba/saves, (sd/usb):/mgba/cheats ...? That would make show them in the ini file.
